packages/experimental-app-router/src/client.ts

```diff
@@ -65,8 +65,8 @@ async function createFaustApolloClient(authenticated = false) {
    * we may set config differently than how we currently do it.
    */
 
-  return new NextSSRApolloClient({
-    cache: new NextSSRInMemoryCache(inMemoryCacheObject),
+  return new ApolloClient({
+    cache: new InMemoryCache(inMemoryCacheObject),
```

Comment:
This was what was blowing up the bundle size:

https://github.com/apollographql/apollo-client-nextjs/issues/95